src/demo/java/org/junitpioneer/jupiter/params/RangeSourcesDemo.java

```diff
@@ -53,6 +45,14 @@ void illegalRange(byte arg) {
 			// since the range will be empty
 		}
 
+		@ParameterizedTest
+		@LongRangeSource(from = 0L, to = 0L, closed = true)
+		void legalRrange(long arg) {
```

Comment:
I wonder if there is a typo here. Should it be spelled `legalRrange` with the extra lowercase "r", or without it?

Reply:
Lol, good catch back to you!